src/Umbraco.Core/Models/Mapping/ContentVariantMapper.cs

```diff
@@ -19,12 +19,12 @@ public ContentVariantMapper(ILocalizationService localizationService, ILocalized
             _localizedTextService = localizedTextService ?? throw new ArgumentNullException(nameof(localizedTextService));
         }
 
-        public IEnumerable<TVariant>? Map<TVariant>(IContent source, MapperContext context) where TVariant : ContentVariantDisplay
+        public IEnumerable<TVariant> Map<TVariant>(IContent source, MapperContext context) where TVariant : ContentVariantDisplay
         {
             var variesByCulture = source.ContentType.VariesByCulture();
             var variesBySegment = source.ContentType.VariesBySegment();
 
-            IList<TVariant>? variants = new List<TVariant>();
+            var variants = new List<TVariant>();
```

Comment:
```suggestion
            List<TVariant> variants = new ();
```
Kinda nitpicky, but i like this new syntax a lot more 👍

Reply:
Time I got used to it I guesss!